plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/authorization/AuthorizationExtension.java

```diff
@@ -207,14 +209,27 @@ public String requestBody(String resolvedExtensionVersion) {
         });
     }
 
+    public AuthenticationResponse getUserRoles(String pluginId, String username, List<SecurityAuthConfig> authConfigs, List<PluginRoleConfig> roleConfigs) {
```

Comment:
Instead of versioning messages. Ideally, we should version the entire extension(e.g elastic extension).